components/micro-gateway-cli/src/main/resources/templates/kubernetesDeployment.mustache

```diff
@@ -9,9 +9,11 @@
     periodSeconds:{{containerConfig.kubernetes.kubernetesDeployment.periodSeconds}}{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.livenessPort}},
     livenessPort:{{containerConfig.kubernetes.kubernetesDeployment.livenessPort}}{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.image}},
     image:"{{containerConfig.kubernetes.kubernetesDeployment.image}}"{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.imagePullPolicy}},
-    imagePullPolicy:"{{containerConfig.kubernetes.kubernetesDeployment.imagePullPolicy}}"{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.env}},
+    imagePullPolicy:"{{containerConfig.kubernetes.kubernetesDeployment.imagePullPolicy}}"{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.imagePullSecrets}},
+    imagePullSecrets: [{{#containerConfig.kubernetes.kubernetesDeployment.imagePullSecrets}}"{{.}}"{{#unless @last}},{{/unless}}{{/containerConfig.kubernetes.kubernetesDeployment.imagePullSecrets}}]{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.env}},
     env:{{{containerConfig.kubernetes.kubernetesDeployment.env}}}{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.buildImage}},
-    buildImage:{{containerConfig.kubernetes.kubernetesDeployment.buildImage}}{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.copyFiles.enable}},
+    buildImage:{{containerConfig.kubernetes.kubernetesDeployment.buildImage}},
```

Comment:
Build image if block is not closed properly and Avoid empty `cmd` attribute
```suggestion
    buildImage:{{containerConfig.kubernetes.kubernetesDeployment.buildImage}}{{/if}}{{#if containerConfig.kubernetes.kubernetesDeployment.cmd}},
```

Reply:
build Image and cmd is inside the if block of build image. cmd does not have any purpose if the build image is disabled

Reply:
And also it should have our default cmd , which executes the gateway shell script rather than the java -jar command